operator-pipeline-images/operatorcert/static_tests/community/bundle.py

```diff
@@ -27,17 +28,66 @@
     validate_timestamp,
 )
 
+# convert table for OCP <-> k8s versions
+# for now these are hardcoded pairs -> if new version of OCP:k8s is released,
+# this table should be updated
+K8S_TO_OCP = {
+    "4.6": "1.19",
+    "4.7": "1.20",
+    "4.8": "1.21",
+    "4.9": "1.22",
+    "4.10": "1.23",
+    "4.11": "1.24",
+    "4.12": "1.25",
+    "4.13": "1.26",
+    "4.14": "1.27",
+    "4.15": "1.28",
+    "4.16": "1.29",
+}
+
 
 class GraphLoopException(Exception):
     """
     Exception raised when a loop is detected in the update graph
     """
 
 
+def process_ocp_version(ocp_metadata_version: Any) -> Any:
+    """
+    Helper function to process the OCP version.
+    OCP version can be either range of mim-max supported versions
+    (v4.7-v4.9), range of up to max supported version (v4.10)
+    or exact single version (=v4.9)
+    """
+    # we are calling indices to extract the supported OCP version
+    # which is then used in operator-sdk bundle validate
+    # to run API deprecation test
+    indices_url = "https://catalog.redhat.com/api/containers/v1/operators/indices"
+    params = {
+        "filter": "organization==community-operators",
+        "sort_by": "ocp_version[desc]",
+        "include": "data.ocp_version",
+        "ocp_versions_range": ocp_metadata_version,
```

Comment:
Given you care only about the latest version you can limit the page size to 1 item here.

Reply:
Also in case the annotation is missing you should exclude the `ocp_versions_range` which gives you the latest supported version.